code/development-env/README.md

```diff
@@ -190,32 +190,19 @@ The value of the cookie can be obtained through a web browser's development tool
 
 ### Run Containers
 
-* Start mongo & vault containers
-
-```bash
-docker-compose -f ./docker/docker-compose-vault.yml -f ./docker/docker-compose-mongo.yml up -d
-```
-
-* Set Vault-App-Role token
-
-```bash
-./scripts/configure-vault.sh
-export VAULT_APP_ROLE= # Set to value output from command on line above
-```
-
 * Start minikube proxy, in separate terminal
 
 ```bash
 kubectl proxy --address 0.0.0.0 --accept-hosts '.*'
 ```
 
-* Start DataLab App, DataLab Api, Infrastructure Api and Auth services.
+* Start Mongo, DataLab App, DataLab Api, Infrastructure Api and Auth services.
 
 ```bash
-docker-compose -f ./docker/docker-compose-vault.yml -f ./docker/docker-compose-mongo.yml -f ./docker/docker-compose-app.yml -f ./docker/docker-compose-proxy.yml up -d
+docker-compose -f ./docker/docker-compose-mongo.yml -f ./docker/docker-compose-mongo.yml -f ./docker/docker-compose-app.yml -f ./docker/docker-compose-proxy.yml up -d
```

Comment:
This seems to call docker-compose-mongo.yml twice?

Reply:
Might also be worth adding --remove-orphans

Reply:
Good spot and good suggestion; I have made the changes